### DISCUSSION TITLE
How can I stop wine from running in the background after close

### DISCUSSION THREAD

**Discussion by IcyAlmond** [cryinkfly/Autodesk-Fusion-360-for-Linux#122] [Q&A] [answered]
After closing fusion 360, it seems like several things in wine keep run running. What's the most effective way of closing them?

**Comment by tinywrkb** [comment, marked as answer]
If it works for you, then use the Flatpak, and then run the `flatpak kill` command.  
Fusion is actually not packaged as a Flatpak, but the installation script is making use of a Wine Flatpak app.  
The means that if you have multiple desktop apps that running through this Wine Flatpak app, then you will need to figure out the instance ID of the Wine Flatpak session that runs Fusion using the `flatpak ps` command, and then kill it with `flatpak kill INSTANCE_ID`.  
If Fusion is the only desktop app that's currently running by the Wine Flatpak app, then just kill it with `flatpak kill org.winehq.flatpak-wine`.

I've been experimenting with Flatpak packaging for Fusion, so I have a dedicated app ID, so I can kill the app with `flatpak kill com.autodesk.fusion360`.

**Reply by cryinkfly** [reply]
With this command you can kill all EXE processes:

    kill -9 $(ps aux |grep -i '\.exe' |awk '{print $2}'|tr '\n' ' ')

**Reply by tinywrkb** [reply]
> With this command you can kill all EXE processes:
> 
> ```
> kill -9 $(ps aux |grep -i '\.exe' |awk '{print $2}'|tr '\n' ' ')
> ```

If you decided to indiscriminately kill every running Wine process, then just kill wineserver.
```
$ wineserver -k
``
